.github/workflows/post-publish-update-version.yml

```diff
@@ -0,0 +1,21 @@
+name: Update application version after release has been published
+
+on:
+  push:
+    branches:
+      - 'update-ga'
```

Comment:
think this needs to be updated

Reply:
yeah, that's the TODO

Reply:
got it. I think we should trigger this on a successful build workflow being completed instead of on push. thoughts?

Reply:
yeah, makes sense, will update